L1Trigger/L1TGlobal/interface/L1TGlobalUtilHelper.h

```diff
@@ -62,7 +62,10 @@ namespace l1t {
                         edm::InputTag const& l1tExtBlkInputTag);
 
     // A module defining its fillDescriptions function might want to use this
-    static void fillDescription(edm::ParameterSetDescription& desc);
+    static void fillDescription(edm::ParameterSetDescription& desc,
```

Comment:
A trivial question: is there a naming convention/guideline for using `fillDescription` or `fillPSetDescription` in utilities used by plugins? (I just know that for plugins one must use `fillDescriptions`)

Reply:
Isn't fillDescription enforced by the base class, while fillPSetDescription is used to call embedded code to let them fill their parameters?

Reply:
`fillDescriptions(edm::ConfigurationDescriptions&)` is required by the framework for modules to use if they want to validate their parameters. Code called from within `fillDescriptions` which deal with a single `edm::ParameterSetDescription` can be named however the developers like. 

For helper classes loaded via the `PluginManager` one can also have them validated via
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#A_Plugin_Module_Using_Another_He

which does depend on the name `fillPSetDescription` to be a static function of the plugin.

In this case, the code is not part of a plugin so the name isn't enforced by the framework. In fact, the Sources also have this sort of pattern for validation and uses `fillDescription` as the name (since it isn't a separate plugin).

https://github.com/cms-sw/cmssw/blob/master/FWCore/Sources/interface/RawInputSource.h#L21